zotero-cli

```diff
@@ -338,7 +338,11 @@ class ZoteroCLI(object):
                         if f in NOTE_FIELDS:
                             d[f] = c.strip()
             if "year" in afields:
-                d['year'] = ZoteroCLI.date(i['data']['date'], i).year
+                dt = i.get('data', {'date': None}).get('date')
```

Comment:
Should be `dt = i.get('data', {}).get('date')` instead...

Reply:
ops! And I was even messing around with the analogous title get. I guess I was just really into typing `date`. thanks for the fix (and the pull)!